internal/controller/deployment_to_pdb_controller.go

```diff
@@ -65,6 +66,24 @@ func (r *DeploymentToPDBReconciler) handleDeploymentReconcile(ctx context.Contex
 
 			// PDB already exists, nothing to do
 			log.Info("PodDisruptionBudget already exists", "namespace", pdb.Namespace, "name", pdb.Name)
+			var pdbWatcher types.PDBWatcher
+			e := r.Get(ctx, req.NamespacedName, &pdbWatcher)
+			if e == nil {
```

Comment:
not found  vs err != nil if its not found you want to skip this part but if its an erro from get you want to return that error

Reply:
Aslo try not to have this much indenting?